package.json

```diff
@@ -42,7 +42,7 @@
   },
   "devDependencies": {
     "@holo-host/chaperone": "git://github.com/Holo-Host/chaperone.git#620c84181efff68ea30eec8e2c3fdd94692c3921",
-    "@holo-host/mock-conductor": "^0.3.0",
+    "@holo-host/mock-conductor": "^0.2.1",
```

Comment:
why did we move back in the version?

Reply:
I just released 0.2.1 yesterday and I don't know where 0.3.0 came from. Prior to my release the master branch had `version: 0.2.0`

Reply:
Just released 0.3.1 to resolve this